autoblocks/_impl/testing/run.py

```diff
@@ -275,26 +279,38 @@ async def async_run_test_suite(
     await global_state.http_client().post(f"{cli()}/end", json=dict(testExternalId=test_id))
 
 
+# Sync fn
+@overload
 def run_test_suite(
     id: str,
     test_cases: Sequence[TestCaseType],
     evaluators: Sequence[BaseTestEvaluator[TestCaseType, OutputType]],
-    fn: Callable[[TestCaseType], Union[OutputType, OutputType]],
```

Comment:
i had meant this to be `Callable[[TestCaseType], Union[OutputType, Awaitable[OutputType]]]` but wouldn't have worked anyway per https://github.com/python/mypy/issues/16646